SamplesCS/Samples/InpaintSample.cs

```diff
@@ -0,0 +1,103 @@
+                "\ts - save the original image, mask image, original+mask image and inpainted image to desktop."
+            );
+
+            using (Mat img0 = Cv2.ImRead(FilePath.Image.Fruits, ImreadModes.AnyDepth | ImreadModes.AnyColor))
+            {
+                using (Mat img = img0.Clone())
+                using (Mat inpaintMask = new Mat(img0.Size(), MatType.CV_8U, 1))
+                using (Mat inpainted = img0.Clone())
+                {
+                    inpainted.SetTo(Scalar.Black);
+                    inpaintMask.SetTo(Scalar.Black);
+
+                    using (Window wImage = new Window("image", WindowMode.AutoSize, img))
+                    {
+                        Point prevPt = new Point(-1, -1);
+                        wImage.SetMouseCallback((MouseEventTypes ev, int x, int y, MouseEventFlags flags, IntPtr userdata)=>
+                        {
+                            if (ev == MouseEventTypes.LButtonUp || (flags & MouseEventFlags.LButton) == 0)
+                            {
+                                prevPt = new Point(-1, -1);
+                            }
+                            else if (ev == MouseEventTypes.LButtonDown)
+                            {
+                                prevPt = new Point(x, y);
+                            }
+                            else if (ev == MouseEventTypes.MouseMove && (flags & MouseEventFlags.LButton) != 0)
+                            {
+                                Point pt = new Point(x, y);
+                                if (prevPt.X < 0)
+                                {
+                                    prevPt = pt;
+                                }
+                                inpaintMask.Line(prevPt, pt, Scalar.White, 5, LineTypes.AntiAlias, 0);
+                                img.Line(prevPt, pt, Scalar.White, 5, LineTypes.AntiAlias, 0);
+                                prevPt = pt;
+                                wImage.ShowImage(img);
+                            }
+                        });
+
+                        for (; ; )
+                        {
+                            switch ((char)Window.WaitKey(0))
+                            {
+                                case (char)27:    // exit
+                                    Window.DestroyAllWindows();
+                                    return;
+                                case 'r':   // restore original image
+                                    inpaintMask.SetTo(Scalar.Black);
+                                    img0.CopyTo(img);
+                                    wImage.ShowImage(img);
+                                    break;
+                                case 'i':   // do Inpaint
+                                case '\r':
+                                    Window wInpaint1 = new Window("inpainted image (algorithm by Alexandru Telea)", WindowMode.AutoSize);

```

Comment:
When Window.Dispose() is called by GC, it close the window unexpectedly. I'll fix it later.